content/pt-br/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -156,5 +156,5 @@ metadata:
 ...
 ```
 
-Utilizando os comandos MySQL, `StatefulSet` e `Service`, você receberá notificação sobre ambos, MySQL e WordPress, o aplicativo maior está incluso.
+No `StatefulSet` e no `Service` do MySQL, você observará que informações sobre ambos, MySQL e WordPress, o aplicativo maior está incluso.
```

Comment:
Minor: comma after "maior" ("o aplicativo maior" sounds to me like it is an explanation for what WordPress is in this context).